src/schema/object_identification.ts

```diff
@@ -161,6 +161,13 @@ export const GlobalIDField = {
   },
 }
 
+export const SingleIDField = {
+  id: {
+    description: "A type-specific ID.",
+    type: GraphQLID,
+  },
+}
+
```

Comment:
We should exclude this field entirely from the v2 schema, as it doesn’t do anything.

Reply:
Done